.github/workflows/pushaction.yml

```diff
@@ -2,9 +2,6 @@ name: Compile and deploy snapshot artifacts
 
 on:
   push:
-    branches:
```

Comment:
I still think it is good to have a restriction on master and branches named sdl/, so please keep this

Reply:
I will force push it out. I need this removed to test the actions as my branch is named `docker`